src/server/network/drain.c

```
@@ -0,0 +1,279 @@
+}
+
+/* --- Message encoders (byte-for-byte identical to the single-message path) --- */
+
+/* Serialize a type 0x32 game message.  reliable=true emits the 5-byte header
+ * [0x32][totalLen][0x80][seqHi][seqLo=0]; reliable=false emits the 3-byte
+ * header [0x32][totalLen][0x00]. */
+static int encode_game(u8 *out, const u8 *payload, int len, bool reliable, u16 seq)
+{
+    if (reliable) {
+        int msg_len = 5 + len;
+        if (msg_len > 255) return 0;
```

Clang (14.0.6)

[warning] statement should be inside braces

(readability-braces-around-statements)